pkg/apis/samples/v1/types.go

```diff
@@ -474,7 +474,7 @@ func (s *Config) ClusterOperatorStatusProgressingCondition(degradedState string,
 		return configv1.ConditionTrue, "", fmt.Sprintf(moving, os.Getenv("RELEASE_VERSION"))
 	}
 	if s.ConditionTrue(RemovePending) {
-		return configv1.ConditionTrue, "", fmt.Sprintf(removing, s.Status.Version)
+		return configv1.ConditionTrue, "", fmt.Sprintf(removing, os.Getenv("RELEASE_VERSION"))
```

Comment:
this change was in `pkg/util/util.go` in PR #220 but that file did not exist in 4.2

Reply:
otherwise, the config.go, handler.go, and handler_test.go deltas match exactly with #220